packages/wdio-allure-reporter/tests/suite.test.ts

```diff
@@ -508,25 +551,31 @@ for (const protocol of ['webdriver', 'devtools']) {
                 outputDir,
             }
             const reporter = new AllureReporter(allureOptions)
+
             reporter.onRunnerStart(runnerStart())
             reporter.onSuiteStart(suiteStart())
             reporter.onTestStart(testStart())
+
             const command = commandStart(protocol === 'devtools')
+
             delete command.body
+
             reporter.onBeforeCommand(command)
             reporter.onAfterCommand(commandEnd(protocol === 'devtools'))
             reporter.onTestSkip(testPending())
             reporter.onSuiteEnd(suiteEnd())
             reporter.onRunnerEnd(runnerEnd())
 
-            const results = getResults(outputDir)
+            const { results } = getResults(outputDir)
+            const requestAttachments = results[0].steps[0].attachments.filter(
+                (attachment: Attachment) => attachment.name === 'Request'
+            )
+
             expect(results).toHaveLength(1)
-            const allureXml = results[0]
-            expect(allureXml('step > name')).toHaveLength(1)
-            expect(allureXml('step > name').eq(0).text()).toEqual(assertionResults[protocol].commandTitle)
-            expect(allureXml('step > title').eq(0).text()).toEqual(assertionResults[protocol].commandTitle)
-            expect(allureXml('test-case attachment[title="Request"]')).toHaveLength(0)
-            expect(allureXml('step').eq(0).attr('status')).toEqual('passed')
+            expect(results[0].steps).toHaveLength(1)
```

Comment:
please assert length before you get value

Reply:
I do that on the line above:

```
expect(results).toHaveLength(1)
```

Reply:
:+1: